pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -166,6 +176,47 @@ func (r *rbac) setDefault() {
 	}
 }
 
+// function to set default concurrency limits for rbac reconciliation
+func (r *rbac) setDefaultConcurrency() {
+	var foundConcurrencyParam = false
+
+	for i, v := range r.tektonConfig.Spec.Params {
+		if v.Name == rbacConcurrencyCountParamName {
+			foundConcurrencyParam = true
+			// If the value set is invalid then set to default value.
+			_, err := strconv.Atoi(v.Value)
+			if err != nil {
+				r.tektonConfig.Spec.Params[i].Value = strconv.Itoa(defaultConcurrencyLimit)
+			}
+			break
+		}
+	}
+	if !foundConcurrencyParam {
+		r.tektonConfig.Spec.Params = append(r.tektonConfig.Spec.Params, v1alpha1.Param{
+			Name:  rbacConcurrencyCountParamName,
+			Value: strconv.Itoa(defaultConcurrencyLimit),
+		})
+	}
+}
+
+func (r *rbac) getNSReconcilationConcurrency() (int, error) {
```

Comment:
What is the puprpose of this, comparingt to the one above, if there a param, we take it in account otherwse we use the default